C `at_auth_cli` showing `APKAM auth failed.` whilst awaiting approval

### Is your feature request related to a problem? Please describe.

`APKAM auth failed.` might be technically correct, but it isn't a friendly/helpful message

### Describe the solution you'd like

Clearer messages saying that the enrollment is awaiting approval would be more meaningful/helpful

### Additional context

This shows (every 10s):

```log
[ERROR] 2025-01-17 14:08:11.750031 | atclient | recv was "error:AT0026:enrollment_id: 00130819-f0ac-4852-b664-8ca31a9d76d3 is pending" and did not have prefix "data:success"
[ERROR] 2025-01-17 14:08:11.750084 | auth_cli | APKAM auth failed. Retrying in 10 secs
```

--- a/packages/atauth/src/wait_for_enrollment.c
+++ b/packages/atauth/src/wait_for_enrollment.c
@@ -21,7 +21,7 @@ int wait_for_enrollment(atclient *ctx, const char *atsign, const atclient_atkeys
     }
 
     if (err_msg != NULL && is_enrollment_denied(err_msg)) {
-      atlogger_log(TAG, ATLOGGER_LOGGING_LEVEL_ERROR, "enrollment id: %s has been denied\n", atkeys->enrollment_id);
+      atlogger_log(TAG, ATLOGGER_LOGGING_LEVEL_DEBUG, "enrollment id: %s has been denied\n", atkeys->enrollment_id);
       ret = 1;
       return ret;
     }

